### DISCUSSION TITLE
Add mass importing  for linux mac and windows

### DISCUSSION THREAD

**Discussion by TigerClips1** [ppy/osu#25684] [Ideas]
so I downloaded 10K  beatmaps and I had to import them one by one it will be easy to add a block of code that lets you import all the beatmaps for Linux Mac and Windows without downloading stable and import all the songs just an idea.

for example, you download a lot of beatmaps in folder A than in osu you can go to folder A and hit a button that import all, osu will go and import all the .osz file format it finds and import it all for you without doing it one by one. 

TigerClips1

**Comment by peppy** [comment]
The game already supports dropping multiple files at once, did you try that?

**Reply by TigerClips1** [reply]
i did not know that thank you
